src/frontend/js/ashley.ts

```diff
@@ -1,7 +1,12 @@
 import * as AshleyEditor from './components/AshleyEditor';
+import { renderEmojis } from './utils/emojis';
 
 // expose some modules to the global window object
 declare var window: any;
 window.Ashley = {
   Editor: AshleyEditor,
 };
+
+document.addEventListener('DOMContentLoaded', async event => {
```

Comment:
Why the callable is `async` ? There is no async code used here

Reply:
Oops, I thought it was the new way to do that :smile: 
I'll remove the `async` part.